docs/howto.rst

```diff
@@ -366,28 +407,35 @@ As there might not be a link between the physiological file and the subject (and
 
 .. code-block:: shell
 
-    phys2bids -in tutorial_file.txt -indir /home/arthurdent/git/phys2bids/phys2bids/tests/data/ -chtrig 1 -ntp 158 -tr 1.2 -outdir /home/arthurdent/physio -heur /home/arthurdent/git/phys2bids/phys2bids/heuristics/heur_tutorial.py -sub 006 -ses 01
+    phys2bids -in tutorial_file.txt -chtrig 1 -ntp 158 -tr 1.2 -outdir physio_bids -heur /home/arthurdent/phys2bids/phys2bids/heuristics/heur_tutorial.py -sub 006 -ses 01
 
 The output will look very similar to our previous calls, when we did not use the ``-heur``, ``-sub`` and ``-ses`` arguments. However, there is one extra line in command line output:
 
 .. code-block:: shell
 
-    INFO:phys2bids.phys2bids:Preparing BIDS output using /home/arthurdent/git/phys2bids/phys2bids/heuristics/heur_tutorial.py
+    INFO       Preparing BIDS output using /home/arthurdent/phys2bids/phys2bids/heuristics/heur_tutorial.pyy
```

Comment:
```suggestion
    INFO       Preparing BIDS output using /home/arthurdent/phys2bids/phys2bids/heuristics/heur_tutorial.py
```

If the `pyy` comes from the log, we need a quick BugFIX PR to correct that.

Reply:
Seems it was my mistake